doc/manual/src/development/building.md

```diff
@@ -34,16 +34,21 @@ $ nix-shell --attr devShells.x86_64-linux.native-clangStdenvPackages
 To build Nix itself in this shell:
 
 ```console
-[nix-shell]$ autoreconfPhase
-[nix-shell]$ ./configure $configureFlags --prefix=$(pwd)/outputs/out
-[nix-shell]$ make -j $NIX_BUILD_CORES
```

Comment:
Does the `ninjaBuildPhase` do anything sufficiently magic in it, or could we mention it, but also include the "manual invocation"? I typically use the Nix devshell like `nix develop .#native-clangStdenvPackages -c fish`, which (maybe obviously) doesn't support these devshell function-type things.

Reply:
Nah it does nothing magic, I was doing `ninja -j20 -k0` until now.